testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/CorruptedLogTestCase.java

```diff
@@ -0,0 +1,217 @@
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.controller.client.helpers.Operations;
+
+import java.net.UnknownHostException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Collectors;
+import java.util.Properties;
+import java.util.stream.Stream;
+import javax.jms.ConnectionFactory;
+import javax.jms.JMSConsumer;
+import javax.jms.JMSContext;
+import javax.jms.JMSProducer;
+import javax.jms.Queue;
+import javax.jms.TextMessage;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import org.apache.commons.lang3.RandomStringUtils;
+
+import org.jboss.arquillian.container.test.api.ContainerController;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.repository.PathUtil;
+import org.jboss.as.test.integration.common.jms.JMSOperations;
+import org.jboss.as.test.integration.common.jms.JMSOperationsProvider;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+import org.jboss.as.test.shared.TimeoutUtil;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ *
+ * @author Emmanuel Hugonnet (c) 2020 Red Hat, Inc.
+ */
+@RunAsClient()
+@RunWith(Arquillian.class)
+public class CorruptedLogTestCase {
+
+    private static final String DEFAULT_FULL_JBOSSAS = "default-full-jbossas";
+
+    @ArquillianResource
+    protected static ContainerController container;
+
+    private ManagementClient managementClient;
+
+    @Before
+    public void setup() throws Exception {
+        if (!container.isStarted(DEFAULT_FULL_JBOSSAS)) {
+            container.start(DEFAULT_FULL_JBOSSAS);
+        }
+        managementClient = createManagementClient();
+        JMSOperations jmsOperations = JMSOperationsProvider.getInstance(managementClient.getControllerClient());
+        setJournalSize(100L);
+        jmsOperations.createJmsQueue("corrupted", "java:jboss/exported/" + "queue/corrupted");
+        jmsOperations.close();
+        managementClient.close();
+        container.stop(DEFAULT_FULL_JBOSSAS);
+        container.start(DEFAULT_FULL_JBOSSAS);
+        managementClient = createManagementClient();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        JMSOperations jmsOperations = JMSOperationsProvider.getInstance(managementClient.getControllerClient());
+        jmsOperations.removeJmsQueue("corrupted");
+        managementClient.getControllerClient().execute(Operations.createUndefineAttributeOperation(jmsOperations.getServerAddress(), "journal-file-size"));
+        managementClient.getControllerClient().execute(Operations.createUndefineAttributeOperation(jmsOperations.getServerAddress(), "journal-max-attic-files"));
+        jmsOperations.close();
+        managementClient.close();
+        container.stop(DEFAULT_FULL_JBOSSAS);
```

Comment:
Now we have two test methods. First call of this method (and the following start in the following setup method) seems to be useless. Can you call this container.stop after all test methods are executed (and not after each test method)? Maybe that ServerSetupTask could help (like in [this example](https://github.com/wildfly/wildfly/blob/master/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/remote/byreference/RemoteInvocationByReferenceTestCase.java#L54))?